frontends/ios/BitBoxApp/BitBoxApp/BitBoxAppApp.swift

```diff
@@ -75,10 +76,44 @@ class GoEnvironment: NSObject, MobileserverGoEnvironmentInterfaceProtocol {
     func setDarkTheme(_ p0: Bool) {
     }
 
-    func systemOpen(_ url: String?) throws {
-        guard let url = URL(string: url!) else { return }
-        if UIApplication.shared.canOpenURL(url) {
-            UIApplication.shared.open(url)
+    // Helper method to get the root view controller
+    private func getRootViewController() -> UIViewController? {
+        guard let scene = UIApplication.shared.connectedScenes
+                .filter({ $0.activationState == .foregroundActive })
+                .first as? UIWindowScene else {
+            return nil
+        }
+
+        return scene.windows.first(where: { $0.isKeyWindow })?.rootViewController
+    }
+
+    func systemOpen(_ urlString: String?) throws {
```

Comment:
The function doesn't throw any exception, I think you can remove the `throws` keyword

Reply:
I don't think we can remove it - this whole class adheres to `MobileserverGoEnvironmentInterfaceProtocol`, i.e. it follows the Environment interface in backend.go. I assume `gomobile` translates a Go error return to `throws`:

https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/c4867b84198ce4dee877b981ebfb0ad1a00a6bf7/backend/backend.go#L150